packages/twenty-server/src/core/messaging/timeline-messaging.service.ts

```diff
@@ -61,7 +61,7 @@ export class TimelineMessagingService {
     const formattedMessageThreads = messageThreads.map((messageThread) => {
       return {
         read: true,
-        senderName: messageThread.last_message_recipient_handle,
+        senderName: messageThread.last_message_participant_handle,
```

Comment:
I feel like this is not exactly a "sender" here because participant could mean anybody? As discussed let's improve that when we move that logic to a list of participants